packages/eslint-plugin/src/rules/index.ts

```diff
@@ -37,5 +38,6 @@ export const rules = {
   "no-single-declare-module": noSingleDeclareModule,
   "no-old-dt-header": noOldDTHeader,
   "no-import-of-dev-dependencies": noImportOfDevDependencies,
+  "npm-naming": npmNaming,
```

Comment:
I can't comment on the file since it wasn't modified, but the actual preset that's exported from this package should filter this entry out of the rules object so that it's not enabled by default.